src/components/atoms/Button/types.ts

```diff
@@ -1,5 +1,17 @@
-export interface Props {
-  isPrimary?: boolean;
-  isDanger?: boolean;
+export type Props = {
```

Comment:
```suggestion
export type ButtonProps = {
```
More accurate type name